.changeset/thirty-days-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/eslint-plugin": patch
```

Comment:
Probably don't need a changeset for changes which involve builds or change sets themselves, as noted by the library authors:

> Not every change requires a changeset

> Since changesets are focused on releases and changelogs, changes to your repository that don't require these won't need a changeset. As such, we recommend not adding a blocking element to contributions in the absence of a changeset.

Going to look into what core does to avoid blocking PRs which don't require changes.

Reply:
Couple options:

1.
- `"changedFilePatterns": []` to scope changes only to our source code, so changesets won't be required for changes to build files, config files, or tests.
- Update the ci-build.yml to check for changes using `npx changeset status --since=main` (may need to adjust for other branches during a release)

2.
Use the GitHub changeset bot: https://github.com/apps/changeset-bot

Reply:
Let's be honest, core is not very good at dealing with changelogs... If your change isn't supposed to be in the changelog, you just leave a blank message. Changesets doesn't support that, and I don't think we should have any requirement at all to have a changeset file in the PR build. In other words, PRs should be merged with or without a changelog entry, regardless of what files were changed. We can always leave comments requesting a changelog entry when we review a PR.

Reply:
Makes sense to me.

Reply:
Indeed - the changes we'll have in this library will be much less numerous than core, so I think just catching in PR review is fine.